pkg/build/helm.go

```diff
@@ -46,6 +47,9 @@ var (
 	// Currently tags are set as `gcr.io/istio-testing` or `gcr.io/istio-release`
 	hubs = []string{"gcr.io/istio-testing", "gcr.io/istio-release"}
 
+	// The helm repo alias we use to point to the actual Helm repository.
+	releaseRepoAlias = "@istio-release"
```

Comment:
TIL...

Reply:
In our docs we do `helm repo add istio https://istio-release.storage.googleapis.com/charts` though so `@istio` may be more appropriate?

Reply:
LGTM just pending decision on ^

Reply:
Yeah I was gonna update the docs either way, and figured `@istio-release` was more descriptive, but I don't have a strong opinion.

Reply:
Actually, we don't even need it, we can keep the `file` refs, it's actually simpler, since `helm package` will pack the subcharts into the tgz anyway.